supermarq-benchmarks/supermarq/benchmarks/bit_code.py

```diff
@@ -73,6 +73,14 @@ def circuit(self) -> cirq.Circuit:
 
         return circuit
 
+    def qiskit_circuit(self) -> qiskit.QuantumCircuit:
```

Comment:
for parity, wdyt about also changing `circuit()` to `cirq_circuit()` in each benchmark subclass? then we could add the following to `supermarq.benchmark.Benchmark` so we wouldn't need to change any tests or break anything:
```python
    @abc.abstractmethod
    def cirq_circuit(self) -> cirq.Circuit | list[cirq.Circuit]:
        """Returns the cirq circuit(s) corresponding to the current benchmark parameters."""

    def circuit(self) ->  cirq.Circuit | list[cirq.Circuit]:
        """Returns the quantum circuit(s) corresponding to the current benchmark parameters."""
        return self.cirq_circuit()
```

Reply:
(alternatively we could just add this to `Benchmark` and leave the subclasses alone:
```python
    def cirq_circuit(self) ->  cirq.Circuit | list[cirq.Circuit]:
        """Returns the cirq circuit(s) corresponding to the current benchmark parameters."""
        return self.circuit()
```
which imo is slightly more awkward in terms of parity but also would be a smaller change and probably fine)

Reply:
Went with option 2 as I started with that initially and seemed to require a smaller overall change